openshift/rh-che.config.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: che
+type: Opaque
+data:
+  che-host: "glusterpoc37aws.devshift.net"
+  workspace-storage: "/home/user/che/workspaces"
+  workspace-storage-create-folders: "false"
+  local-conf-dir: "/etc/conf"
+  openshift-serviceaccountname: "che"
+  che-server-evaluation-strategy: "always-external-custom"
+  che.logs.dir: "/data/logs"
+  che.docker.server_evaluation_strategy.custom.template: "<serverName>-<if(isDevMachine)><workspaceIdWithoutPrefix><else><machineName><endif>-<if(workspacesRoutingSuffix)><workspacesRoutingSuffix><else><externalAddress><endif>"
```

Comment:
I think you can simplify this to:
```
<serverName>-<if(isDevMachine)><workspaceIdWithoutPrefix><else><machineName><endif>-<workspacesRoutingSuffix>
```
Since in the multi-tenant scenario, the workspaceRoutingSuffix is always provided.

Reply:
ok, lemme try